src/Microsoft.Health.Dicom.Azure/Microsoft.Health.Dicom.Azure.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Implementation of Azure dependencies used in DICOM core library</Description>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
nit: I don't think we want implicit usings. But we could try `nullable` here lol

Reply:
Nullable is painful. I can remove implicit